pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -224,25 +263,51 @@ def newpkgcb(pkgId, name, arch):
         packages_pb.save()
         erratum_pb.save()
 
+        remote_url = self.new_url or self.remote.url
+
         with ProgressBar(message='Downloading Metadata Files') as metadata_pb:
             downloader = self.remote.get_downloader(
-                url=urljoin(self.remote.url, 'repodata/repomd.xml')
+                url=urljoin(remote_url, 'repodata/repomd.xml')
             )
             # TODO: decide how to distinguish between a mirror list and a normal repo
             result = await downloader.run()
             metadata_pb.increment()
 
+            if self.kickstart:
+                d_artifacts = []
+                for path, checksum in self.kickstart["download"]["images"].items():
```

Comment:
Will this part also sync the part of [this .treeinfo file](http://download-ipv4.eng.brq.redhat.com/released/RHEL-7/7.3/Server/s390x/os/.treeinfo) that has:

```
[stage2]
mainimage = images/install.img
```

Reply:
yes, it does sync this part:
https://github.com/fabricio-aguiar/pulp_rpm/blob/5202-kickstart-sync/pulp_rpm/app/tasks/utils.py#L170-L174
but it will only stay in `distribution_tree._artifacts` because I don't create an `Image` form them if it does not appear on the `checksums` section.